CHANGES.rst

```diff
@@ -0,0 +1,13 @@
+Changes
+=======
+
+This is a record of all past ``gettsim`` releases and what went into them in reverse
+chronological order. We follow `semantic versioning <https://semver.org/>`_ and all
+releases are available on `Anaconda.org <https://anaconda.org/gettsim/gettsim>`_.
+
+0.0.1 - 2019-
```

Comment:
* Most code written by Eric Sommer based on IZAΨMOD (link to https://www.iza.org/publications/dp/8553/documentation-izapsmod-v30-the-iza-policy-simulation-model) 
* Many updates and interface changes by Maximilian Blesch, Janos Gabler, Tobias Raabe, Hans-Martin von Gaudecker
* Setup of development workflow by Tobias Raabe. 

[link GitHub usernames]